src/index.js

```diff
@@ -96,19 +96,19 @@ export default class ServerlessGitVariables {
     }
 
     for (const functionName of this.serverless.service.getAllFunctions()) {
-      const func = this.serverless.service.getFunction(functionName);
```

Comment:
I think you still want this semicolon.  At least, if the other `const` declarations need one, this would need one, too.